core/relay/src/db.rs

```diff
@@ -7,6 +7,8 @@ use crate::error::Error;
 /// Struct for managing all db interactions
 pub struct Database {
     pub db_path: String,
+    pub db_open_max_retries: u32,
+    pub db_open_retry_backoff_time: u32,
```

Comment:
Convention is generally to have `sec`, `msec`, etc. here, so either replace `time` with `msec` or just add `_msec` at the end.